File: ReactNative/src/modules/todo/services/update-todo.service.ts
import { ITodoForm } from '../../../@types/todo-form';
import { TokenAxios } from '../../common/utils/token-axios';
import { TODO_URL } from '../../common/utils/url';

export async function requestUpdateTodo({
  idx,
  memo,
}: ITodoForm): Promise<void> {
  const AXIOS = await TokenAxios();
  await AXIOS.put(`${TODO_URL}/${idx}`, {
    memo: memo,
  });
}

File: flutter_todo/lib/src/modules/todo/services/update_todo_service.dart
import 'package:flutter_todo/src/modules/common/utils/token_dio.dart';
import 'package:flutter_todo/src/modules/common/utils/url.dart';

Future<void> requestUpdateTodo(int idx, String memo) async =>
    (await tokenDio()).put('$todoUrl/$idx', data: {
      "memo": memo,
    }).catchError((error) => throw error);
